scripts/50_update_files.sh

```diff
@@ -142,8 +150,14 @@ if [ ! -b "${TARGET_STORAGE_DEVICE}" ]; then
     exit 1
 fi
 
-if [ ! -f "${SYSTEM_UPDATE_CONF_DIR}/${UPDATE_EXCLUDE_LIST_FILE}" ]; then
-    echo "Update failed: file '${SYSTEM_UPDATE_CONF_DIR}/${UPDATE_EXCLUDE_LIST_FILE}' not found."
+if [ ! -d "${SYSTEM_UPDATE_CONF_DIR}" ]; then
```

Comment:
I do not understand why there are multiple locations here and above to loop for filter files?